ets-demo/etsdemo/tests/test_app.py

```diff
@@ -144,3 +148,87 @@ def test_init_dict_used_by_children(self):
             example.run_code()
             self.assertIn("CONSTANT", example.locals)
             self.assertIn("HasTraits", example.locals)
+
+
+class TestParseSource(unittest.TestCase):
```

Comment:
This test is copied from `traitsui/tests/test_extras`. We can reorganize functions and modules later.